contracts/modules/Checkpoint/Voting/APLCR/AdvancedPLCRVotingCheckpoint.sol

```diff
@@ -602,6 +608,9 @@ contract AdvancedPLCRVotingCheckpoint is AdvancedPLCRVotingCheckpointStorage, Vo
      * @return address[] list of invesotrs who are remain to vote
      */
     function getPendingInvestorToVote(uint256 _ballotId) external view returns(address[] memory pendingInvestors) {
+        // @TODO for clarity, shall we revert instead of returning an empty array?
+        // if so, can we change it into a modifier?
```

Comment:
For the ease of handling on the dApp side, we try to return 0 values instead of revert error @VictorVicente @FahdW what do you prefer?

Reply:
Makes sense. Let's stick to returning empty arrays.